app/repositories/onlinetesting/OnlineTestRepository.scala

```diff
@@ -223,6 +223,15 @@ trait OnlineTestRepository extends RandomSelection with ReactiveRepositoryHelper
     findAndUpdateCubiksTest(cubiksUserId, update, ignoreNotFound = true)
   }
 
+  def updateTestCompletionTime(orderId: String, completedTime: DateTime) = {
```

Comment:
We already have this method - should we not use instead:
  def updateTestCompletionTime2(orderId: String, completedTime: DateTime) = {
    import repositories.BSONDateTimeHandler
    val update = BSONDocument("$set" -> BSONDocument(
      s"testGroups.$phaseName.tests.$$.completedDateTime" -> Some(completedTime)
    ))

    findAndUpdatePsiTest(orderId, update, ignoreNotFound = true)
  }